packages/canary-web-components/src/components/ic-tree-item/ic-tree-item.tsx

```diff
@@ -33,7 +33,7 @@ let treeItemIds = 0;
   shadow: true,
 })
 export class TreeItem {
-  private treeItemId = `ic-tree-item-${treeItemIds++}`;
+  private defaultTreeItemId = `ic-tree-item-${treeItemIds++}`;
```

Comment:
Could this just become the default value for the prop, rather than its own private variable?

Reply:
I tried this but it made the docs for the prop confusing. It states the default value for the prop is `ic-tree-item-${treeItemIds++}` but it is not clear that treeItemIds starts at 0 internally unless the developer looks into the code.